main/inc/lib/main_api.lib.php

```diff
@@ -355,6 +355,13 @@
 define('CATALOG_SESSIONS', 1);
 define('CATALOG_COURSES_SESSIONS', 2);
 
+//Hook
```

Comment:
Some documentation about what those constants means.
Maybe change the name to something more descriptive? PRE or POST update?